### DISCUSSION TITLE
MultiVendor Shipping Setup Options

### DISCUSSION THREAD

**Discussion by gregkeys** [inspireui/support#5487] [New Feature/Ideas Requests]
A vendor should be able to setup shipping options in the app, they should be able to select if they do local delivery, configure delivery fees, set shipping costs such as by weight, class and flat rate shipping.

**Comment by creationwebuk** [comment]
Yes this needs to be added, you have to go all the way to the website to setup shipping. Can't even do it in the vendor admin extension app either

**Comment by ElsaidMahmous** [comment]
Up

**Comment by Mokwa123** [comment]
This is needed please on the fluxstore manager app (Native)

**Comment by byteknight1** [comment]
A must have for vendor management.

**Comment by Yan-Aleman** [comment]
Support ticket plz #5440, #5351

**Comment by DivitpatelMumbai** [comment]
This feature must be available!

**Comment by Edmonts** [comment]
❤️❤️❤️ Good idea, let's wait for its availability.

**Comment by NextClouds** [comment]
Available?

**Comment by Yan-Aleman** [comment]
Plz support #5656, a very important for vendors, the amount of delivery zone 🙂🚀

**Comment by mohammedkezzh** [comment]
Yes, this is a significant moment for the food delivery business. Vendors should have a way to set free shipping above a specified amount, and paid shipping up to a threshold order amount. To different zones.  GOOD ticket!

**Comment by vvvSayle** [comment]
Yes, please do it!
